src/plugins/ResizeReorder/ResizeReorderCell.js

```diff
@@ -0,0 +1,402 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ResizeReorderCell
+ * @typechecks
+ */
+
+import FixedDataTableCellDefault from 'FixedDataTableCellDefault';
```

Comment:
Any tips on how to best review this file?

Reply:
This component is based on the `FixedDataTableCell` component before the resize/reorder logic was stripped off. 

It maintains state
* isColumnReordering: if true the user is currently reordering the cell
* displacement: how much the cell will be translated while reordering
* reorderingDisplacement: seems unused
* columnReorderingData: information about the reordering (I think this can be simplified down)

Besides rendering the children (actual cell), it also renders two different components `ResizerKnob` and `ReorderHandle` which makes up the UX that handles resizing and reordering respectively. These are somewhat strongly tied to the the state defined above.